sdk/php/src/Tests/Connection/ProcessSessionDaggerConnectionTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace DaggerIo\Tests\Connection;
```

Comment:
So good to see these!

Can you please add them to our CI, e.g.
- https://github.com/dagger/dagger/blob/main/internal/mage/sdk/python.go 
- https://github.com/dagger/dagger/blob/bdaefc4d9769c949618b70e521aa4094e58d50b2/.github/workflows/sdk-python.yml